messages/tree.export.md

```diff
@@ -30,7 +30,7 @@ Directory in which to generate the JSON files; default is current directory.
 
 - Export records retrieved with the specified SOQL query into a single JSON file in the current directory; the command uses your default org:
 
-  <%= config.bin %> <%= command.id %> --query "SELECT Id, Name, (SELECT Name, Address__c FROM Properties__r) FROM Broker__c"
```

Comment:
bump dev-scripts to avoid linting the .md files?

Reply:
aww, it's on current dev-scripts.  I don't know why that's still happening.  Maybe something in vscode ext?